docs.json

```diff
@@ -776,131 +797,9 @@
-                          "api/resources/tls-edge-mutual-tls-module",
-                          "api/resources/tls-edge-tls-termination-module",
-                          "api/resources/tls-edge-traffic-policy-module"
-                        ]
-                      },
-                      {
-                        "group": "Backends",
-                        "pages": [
-                          "api/resources/failover-backends",
-                          "api/resources/http-response-backends",
-                          "api/resources/tunnel-group-backends",
-                          "api/resources/weighted-backends"
-                        ]
-                      }
-                    ]
-                  }
-                ]
-              },
-              {
-                "group": "Traffic Policy",
-                "pages": [
-                  "api/resources/certificate-authorities",
-                  {
-                    "group": "IP",
-                    "pages": [
-                      "api/resources/ip-policies",
-                      "api/resources/ip-policy-rules"
-                    ]
-                  },
-                  {
-                    "group": "Applications",
-                    "pages": [
-                      "api/resources/application-users",
-                      "api/resources/application-sessions"
-                    ]
-                  }
-                ]
-              },
-              {
-                "group": "Secure Tunnels",
-                "pages": [
-                  "api/resources/agent-ingresses",
-                  "api/resources/tunnels",
-                  "api/resources/tunnel-sessions"
-                ]
-              },
-              {
-                "group": "Observability",
-                "pages": [
-                  "api/resources/event-destinations",
-                  "api/resources/event-sources",
-                  "api/resources/event-subscriptions"
-                ]
-              },
-              {
-                "group": "IAM",
-                "pages": [
-                  "api/resources/ip-restrictions",
-                  "api/resources/api-keys",
-                  "api/resources/ssh-credentials",
-                  "api/resources/credentials",
-                  "api/resources/bot-users"
-                ]
-              },
-              {
-                "group": "SSH Certificates",
-                "pages": [
-                  "api/resources/ssh-certificate-authorities",
-                  "api/resources/ssh-host-certificates",
-                  "api/resources/ssh-user-certificates"
-                ]
-              },
-              {
-                "group": "Partners",
-                "pages": ["api/resources/abuse-reports"]
+                "group": "Reference",
+                "openapi": "https://raw.githubusercontent.com/ngrok/ngrok-openapi/refs/heads/main/ngrok.yaml"
```

Comment:
This is the primary change in this file. Sorry about the formatting changes, my IDE early auto-formatted the file. I can revert those changes if you like.